container/container_contract.go

```diff
@@ -357,6 +358,7 @@ func Delete(containerID []byte, signature interop.Signature, token []byte) {
 	}
 	removeContainer(ctx, containerID, ownerID)
 	runtime.Log("remove container")
+	runtime.Notify("DeleteSuccess", interop.Hash256(containerID), signature, token)
```

Comment:
Why hash container ID?

Reply:
It is a type case, to have proper argument type in the manifest.